src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -523,6 +525,45 @@ RequestContext<object> requestContext
             await requestContext.SendResult(result.First().ToString());
         }
 
+        private async Task HandleGetCommandRequest(
+            string param,
+            RequestContext<object> requestContext)
+        {
+            PSCommand psCommand = new PSCommand();
+            if (!string.IsNullOrEmpty(param)) {
+                psCommand.AddCommand("Microsoft.PowerShell.Core\\Get-Command").AddArgument(param);
+            }
+            else
+            {
+                psCommand.AddCommand("Microsoft.PowerShell.Core\\Get-Command")
+                        .AddParameter("CommandType", new[]{"Function", "Cmdlet", "ExternalScript"})
+                    .AddCommand("Select-Object")
+                        .AddParameter("Property", new[]{"Name", "ModuleName"})
+                    .AddCommand("Sort-Object")
```

Comment:
Ah. Totally my fault on this one... We should add the namespace to this command and the command two lines up.

Not sure what it is off the top of my head. Check by doing:

```powershell
Get-Command Sort-Object
```

Reply:
As much as I would love to let you take the fall for this one, when I was typing it out I looked at it and thought 🤔 "These should probably be fully qualified" 🤔 
But then I though "Nah forget it! Yo homes to Bel Air!"
🎵  "I pulled up to a PR about 7 or 8..."


FWIW: both `Sort-Object` and `Select-Object` are in `Microsoft.PowerShell.Utility`